js/classes/entities/midground/npc/village/papa_chad.js

```diff
@@ -52,24 +52,21 @@ class PapaChad {
 
     /** Change what Papa Chad is doing and where it is. */
     update() {
```

Comment:
Like you said during our meeting there is some clunky stuff going on in here to update the story/react to story updates, but since it works I don't think we need to care too much. :)